neo/SmartContract/ApplicationEngine.cs

```diff
@@ -229,6 +233,23 @@ private bool CheckStackSize(OpCode nextInstruction)
             return true;
         }
 
+        private bool CheckDynamicInvoke(OpCode nextInstruction)
+        {
+            if(nextInstruction == OpCode.APPCALL || nextInstruction == OpCode.TAILCALL)
+            {
+                for (int i = CurrentContext.InstructionPointer + 1; i < CurrentContext.InstructionPointer + 20; i++) 
```

Comment:
if (CurrentContext.InstructionPointer + 20 > CurrentContext.Script.Length) ?

Reply:
Not sure if we need this check.  If we go beyond the end of the script, the error will be caught in the `Execute` method